patches/server/0576-Add-support-for-hex-color-codes-in-console.patch

```diff
@@ -243,7 +244,11 @@ index 0000000000000000000000000000000000000000..b4d0b7ecd56ab952319946854168c129
 +        while (matcher.find()) {
 +            int format = LOOKUP.indexOf(Character.toLowerCase(matcher.group().charAt(1)));
 +            if (format != -1) {
-+                matcher.appendReplacement(buffer, ansi ? ansiCodes[format] : "");
++                String replacement = "";
++                if (ansi) {
++                    replacement = format < 16 ? ANSI_RESET + ansiCodes[format] : ansiCodes[format]; // handle the upstream quirk at CraftChatMessage L115
```

Comment:
idk if this comment is really needed, it's a quirk of the format in general not just CB's impl (which we don't actually use for logging iirc, we use the LegacyComponentSerializer)

Reply:
you're right i will change this